src/functionalTest/resources/environment.conf

```diff
@@ -9,7 +9,7 @@ serviceBus {
   namespaceConnectionString = ${TEST_SERVICE_BUS_CONNECTION_STRING}
   topic = ${TEST_SERVICE_BUS_TOPIC}
   subscription = ${TEST_SERVICE_BUS_SUBSCRIPTION}
-  pollingDelayInMs = ${TEST_SERVICE_BUS_POLLING_DELAY_MS}
-  messageLockTimeoutInMs = ${TEST_SERVICE_BUS_MESSAGE_LOCK_TIMEOUT_MS}
+  pollingDelayInMs = 30000
+  messageLockTimeoutInMs = 30000
```

Comment:
Polling delay is fine, as it's a property of the service. Message lock timeout is a property of the subscription, set in terraform - it would be best if it came from there.